packages/shared/useIntervalFn/demo.vue

```diff
@@ -7,7 +7,7 @@ const word = ref('Hello')
 const interval = ref(500)
 
 const { pause, resume, isActive } = useIntervalFn(() => {
-  word.value = greetings[Math.round(Math.random() * (greetings.length - 1))]
+  word.value = greetings[Math.floor(Math.random() * greetings.length)]
```

Comment:
or maybe we can `import { rand } from '@vueuse/shared` first, and:
```suggestion
  word.value = greetings[rand(0, greetings.length - 1)]
```

Reply:
Done!